### PR TITLE
Lerna with workflow icons

### PR DESCRIPTION
**Pull request by jianliao**
## Description
Switch from private @spectrum/spectrum-icons package to open sourced @adobe/spectrum-css-workflow-icons package.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: tools/bundle-builder/index.js [hunk 1]
```diff
@@ -112,8 +112,9 @@ function copyIcons() {
   // lerna bug: require.resolve() won't find spectrum-icons within spectrum-css-compat, so we have to assume it's at the top level
   // note: maybe it needs a package.main?
   return gulp.src([
-    `${dirs.topLevel}/node_modules/@spectrum/spectrum-icons/dist/svg/**`,
-    `${dirs.topLevel}/node_modules/@spectrum/spectrum-icons/dist/lib/**`
+    `${dirs.topLevel}/node_modules/@adobe/spectrum-css-workflow-icons/dist/**`,
+    `!${dirs.topLevel}/node_modules/@adobe/spectrum-css-workflow-icons/dist/18/**`,
```
Comment (lazd) [inline tools/bundle-builder/index.js]: `dist/**` will cover all files and folders within `dist`, so the second two globs should not be necessary.

Comment (jianliao) [reply tools/bundle-builder/index.js]: But that two globs is to exclude the separated SVG icons to be copied to spectrum-css dist folder. Do we want to release them with spectrum-css?

Comment (lazd) [reply tools/bundle-builder/index.js]: OH. No, we don't want to release them, thanks!

**Comment by lazd** [comment]
@jianliao will wait on this PR until color icons are in. Thanks!

**Comment by jianliao** [comment]
@lazd Color icons were in.

**Comment by jianliao** [comment]
Closed as lerna branch was merged to master branch. Will create another PR against master branch.

**Pull request closed without merging**